aardwolf-templates/Cargo.toml

```diff
@@ -26,8 +26,8 @@ path = "../aardwolf-types"
 version = "0.1"
 path = "../aardwolf-models"
 
-[dev-dependencies]
-ructe = "0.16"
+#[dev-dependencies]
+#ructe = "0.16"
```

Comment:
Can you delete this commented out code before merging?

Reply:
Done.
Ructe was used to compile the old-style templates, and I am really not interested in making them work anymore.